.github/workflows/publish.yml

```diff
@@ -47,7 +48,8 @@ jobs:
           cd dist && echo "hashes=$(sha256sum * | base64 -w0)" >> $GITHUB_OUTPUT
 
       - name: "Upload dists"
-        uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4.3.3
+        uses: actions/upload-artifact@4.6.2 # immutable release
```

Comment:
Can you please stick to the existing scheme? There may be an argument to switch back to version numbers, but I'd rather do this separately, for all actions, and make sure that the format will be recognized by dependabot.